views/macros/input-text.njk

```diff
@@ -7,9 +7,9 @@
     {% set firstError = errors[errors.keys()[0]] %}
 {% endif %}
 
-{% macro textInput(name, label, attributes) %}
+{% macro textInput(name, label, attributes, interpolation={}) %}
```

Comment:
poking at the basic macros from `node-starter-app` makes me a bit scared for the future but there's probably not a better way to do this.

Reply:
Yeah there should be a supported way to do this, maybe a future PR to the starter repo.